### DISCUSSION TITLE
Rounded table, problem with bottom corners

### DISCUSSION THREAD

**Discussion by michelinok** [twbs/bootstrap#39405] [Q&A] [answered]
Hi!
I'm still learning bootstrap....so don't kill me 😄 
I'm trying to get round corners around a table, my code seems to works ok but i get a little line on the bottom of the table

Here's my code:

```
<div class="container" style="max-width: 300px">
<table class="table rounded-5 overflow-hidden">
  <thead>
  	<tr>
		<th colspan="3" class="text-center">Larry the Bird</th>
    </tr>
  </thead>
  <tbody>
    <tr class="table-warning">
      <td>Mark</td>
      <td class="text-end">@mdo</td>
    </tr>
    <tr class="table-danger">
      <td>Jacob</td>
      <td class="text-end">Thornton</td>
    </tr>
    <tr class="table-primary">
	  <td>Thornton</td>
      <td class="text-end">@twitter</td>
    </tr>
  </tbody>
</table>
</div>

```

Any help is really really appreciated!!!!!
I'm still a newbie...still learning, maybe the problem is very stupid.

![Screenshot 2023-11-14 212828](https://github.com/twbs/bootstrap/assets/1754401/3a76e0e1-534a-4e71-bf2d-85b6cc94b415)

**Comment by julien-deramond** [comment, marked as answer]
Hey @michelinok 
Based on your current code, considering that each `<td>` has a bottom border by default in Bootstrap, you could add a `.border-bottom-0` to each `<td>` of the last row, or by CSS with something like this:

```css
tr:last-of-type > td {
  border-bottom-width: 0;
}
```

There are probably other ways to do that, but that's one solution :)

**Reply by michelinok** [reply]
You saved mi life!
Many thanks!!!!!
